scss/components/alert.scss

```diff
@@ -0,0 +1,52 @@
+@import "../core/settings";
+@import "../core/mixins";
+@import "../core/functions";
+
+/*!
+.tn-alert+(--notification. --error)
+    .tn-alert__action
+*/
+$block: ns(alert);
+
+.#{$block} {
+    //SETTINGS
+    $tn-alert-background-color: tn-color(text, 3) !default;
+    $tn-alert-background-color--notification: tn-color(status, 2) !default;
+    $tn-alert-background-color--error: tn-color(status, 3) !default;
+    $tn-alert-color: tn-color(text-inverse, 1) !default;
+    $tn-alert-border-color: tn-color(neutral, 2) !default;
+    $tn-alert-height: tn-space(13) !default;
+
+    // Block
+    color: $tn-alert-color;
+    border: solid 1px $tn-alert-border-color;
+    background-color: $tn-alert-background-color;
+    height: $tn-alert-height;
```

Comment:
Should be min-height ... always test variable content length.

<img width="755" alt="screen shot 2017-08-31 at 2 38 06 pm" src="https://user-images.githubusercontent.com/424119/29942034-1228d5e6-8e5a-11e7-9ab5-03dedc9c0654.png">